src/solution.c

```diff
@@ -102,6 +105,7 @@ void solution_send_nmea(gnss_solution *soln, dops_t *dops,
   );
 }
 
+/* definition of "flags" - 1 if float, 0 if fixed*/
```

Comment:
Perhaps this is a good time to write a Doxygen style comment for this function.

See: https://swiftnav.hackpad.com/Engineering-Playbook-fKcJhOl85sD#:h=C